compiler/sizealignoffsetimpl.nim

```diff
@@ -205,7 +208,11 @@ proc computeSizeAlign(conf: ConfigRef; typ: PType) =
   assert typ != nil
   let hasSize = typ.size != szUncomputedSize
   let hasAlign = typ.align != szUncomputedSize
-
+  let acyclic = tfAcyclic in typ.flags
+  if acyclic:
+    typ.size = szUnknownSize
+    typ.align = szUnknownSize
+    return
```

Comment:
Whether a type is acyclic or not has nothing to do with its size/alignment properties.

Reply:
maybe I need understand how the size compute first before I write these code.